client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala

```diff
@@ -89,6 +89,9 @@ object ConfUtils {
     "spark.delta.sharing.oauth.tokenRenewalThresholdInSeconds"
   val OAUTH_EXPIRATION_THRESHOLD_SECONDS_DEFAULT = 10 * 60 /* 10 mins */
 
+  val NEVER_USE_HTTPS = "spark.delta.sharing.network.never.use.https"
```

Comment:
nit: shall we we have a better name? or you prefer `never`?

Reply:
this naming mirrors existing settings (like [this one](https://src.dev.databricks.com/databricks/runtime@5268038c448df2bc3226fc11b80ce37319d3a9c1/-/blob/core/src/main/scala/com/databricks/io/HadoopConfUtils.scala?L61))